tests/performance/test_performance.py

```diff
@@ -33,52 +31,126 @@ def get_fqn(obj):
     return f'{obj.__module__}.{obj.__name__}'
 
 
-TEST_CASES_PATH = pathlib.Path(__file__).parent / 'test_cases'
-TEST_CASES_PATH_LEN = len(str(TEST_CASES_PATH)) + 1
-TEST_CASES = [str(test_case) for test_case in TEST_CASES_PATH.rglob('*.json')]
-IDS = [test_case[TEST_CASES_PATH_LEN:] for test_case in TEST_CASES]
+def _get_all_dataset_generators():
+    """Get a list of all dataset generators."""
+    generators = []
 
+    for generator in BaseDatasetGenerator.__subclasses__():
+        generators.extend(generator.__subclasses__())
 
-@pytest.mark.parametrize('config_path', TEST_CASES, ids=IDS)
-def test_performance(config_path):
-    """Run the performance tests for RDT.
+    return generators
+
+
+def _build_transformer_map():
+    """Build a map of data type to transformer.
+
+    Output:
+        dict:
+            A mapping of data type (str) to a list of transformers.
+    """
+    transformers = BaseTransformer.get_subclasses()
+    transformers_map = {}
+
+    for transformer in transformers:
+        input_type = transformer.get_input_type()
+        input_type_transformers = transformers_map.get(input_type, [])
+        input_type_transformers.append(transformer)
+        transformers_map[input_type] = input_type_transformers
```

Comment:
minor: you could make `transformers_map` a `default_dict`. So these lines can be changed to something like
```
transformers_map = defaultdict(list)
transformers_map[input_type].append(transformer)
```

Also , there will be a method exactly like this in the `transformers.__init__.py` called `get_transformers_by_type`

Reply:
Re-using the `get_transformers_by_type` method sounds good to me. I'll wait for that PR.